Tools/autotest/apmrover2.py

```diff
@@ -689,6 +689,18 @@ def autotest(self):
 
             self.run_test("Test RC overrides", self.test_rc_overrides)
 
+            self.wait_ready_to_arm()
+            self.mavproxy.send('rc 3 1500\n')  # throttle at zero
+            self.arm_vehicle()
+            self.set_parameter("FS_GCS_ENABLE", 0)
+            self.mavproxy.send('mode guided\n')
```

Comment:
This should be done in the test.  Make them self-contained so that we can run them stand-alone in the future.